apps/full-site-editing/full-site-editing-plugin/wpcom-block-editor-nux/src/store.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { plugins, registerStore, use } from '@wordpress/data';
+
+use( plugins.persistence, {} );
```

Comment:
Im a little confused how this is working and what it is doing for us.  It reads to me like we are inheriting persistence functionality onto an empty object.  How does this effect what we end up registering with `registerStore` (which seems to be using `persist: true` anyways) ?

Reply:
Well pointed out, @Addison-Stavlo!

I'm not super clear on how the persistence works, and the Guten readme got me confused.
I've tried removing this line, and it seems it's persisting as good as before.

Apparently, `use( plugins.persistence, {} );` creates a new localStorage item, which in this case contained only the `automattic/nux` state.
But, the `persist: true` added the `automattic/nux` state to the default WP localStorage, so it was indeed plenty enough.